computer_vision/noise_augmentation.py

```diff
@@ -0,0 +1,99 @@
+import glob
+import os
+import random
+from string import ascii_lowercase, digits
+
+import cv2
+import numpy as np
+
+# Params
+IMAGE_DIR = "1"
+OUTPUT_DIR = "out"
+NOISE_TYPE = "salt-and-pepper"  # ("salt-and-pepper" or "gaussian")
+ALPHA = 0.1  # ("salt-and-pepper" amount or "gaussian" variance)
+
+
+def main() -> None:
+    """
+    Get images list from input dir.
+    Update new images.
+    Save images in output dir.
+    """
+    img_paths = get_dataset(IMAGE_DIR)
+    print("Processing...")
```

Comment:
```suggestion
```
These algorithms just return data instead of printing outputs, so this isn't particularly necessary